notebook/static/notebook/less/cell.less

```diff
@@ -21,6 +21,11 @@ div.cell {
             border-color: transparent;
         }
     }
+    
+    &.marked {
```

Comment:
Right now the 3px L border causes the rest of the cell to jump by 2px when it is marked. I think a simple negative margin when the cell is marked should fix this.